recipes/r-rlemon/meta.yaml

```
@@ -0,0 +1,86 @@
+{% set version = '0.2.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rlemon
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/rlemon_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/rlemon/rlemon_{{ version }}.tar.gz
+  sha256: 4a18fa034f197c68daf48daf25c0e41f1b8acbe71d030c6bc1f55e3062a10375
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'  # [win]
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - m2w64-libwinpthread-git          # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-rcpp >=1.0.5
+  run:
+    - r-base
+    - m2w64-libwinpthread-git      # [win]
```

Use the jinja variable:
```suggestion
    - {{ native }}libwinpthread-git  # [win]